### PR TITLE
[Doc] Updated API Compatibility documentation with latest info

### PR DESCRIPTION
**Pull request by Vijay-Nirmal**
- [X] Updated API Compatibility doc with missed out commands
- [X] Add documentation for already existing commands but not documented
- [X] Updated comments in API Compatibility doc to indicate if there is anything missing

Open Question:
1. Should I add commands from the Redis module in API Compatibility doc? Note: ICYMI, [Redis 8 will add most modules to the base image itself.](https://github.com/redis/redis/releases/tag/8.0-m01)

### CODE REVIEW

**Comment by badrishc** [comment]
> Should I add commands from the Redis module in API Compatibility doc?

Thanks for your contributions! There is no need to list modules in there at this point, IMO. We can revisit that later.

**Comment by tmikaeld** [comment]
`ZREVRANGEBYSCORE` was also completed.

**Comment by Vijay-Nirmal** [comment]
@tmikaeld Thanks for letting me know, updated it as part of this PR

File: website/docs/commands/api-compatibility.md [hunk 1]
```diff
@@ -171,53 +230,58 @@ Note that this list is subject to change as we continue to expand our API comman
...
-|  | LATENCY LATEST | ➖ |  |
-|  | [LATENCY RESET](server.md#latency-reset) | ➕ |  |
-|  | LOLWUT | ➕ |  |
-|  | MEMORY DOCTOR | ➖ |  |
-|  | MEMORY MALLOC-STATS | ➖ |  |
-|  | MEMORY PURGE | ➖ |  |
-|  | MEMORY STATS | ➖ |  |
-|  | [MEMORY USAGE](server.md#memory-usage) | ➕ |  |
-|  | MODULE | ➖ |  |
+|  | [LOLWUT](server.md#lolwut) | ➕ |  |
```
Comment (badrishc) [inline website/docs/commands/api-compatibility.md:284]: i don't think Garnet has support for the LOLWUT command. Need to remove here and the description section for it.

**Comment by Vijay-Nirmal** [comment]
@badrishc My bad. I had noted down in my notes wrongly. It was supposed to be `MONITOR` (command below `LOLWUT`), which was already implemented. I have made the changes accordingly.

**Review by badrishc — APPROVED** [review]

**Pull request merged**